### PR TITLE
[AIRFLOW-6473] Show conf in response of dag_state cli command

### PR DESCRIPTION
**Pull request by tooptoop4**
---
Issue link: [AIRFLOW-6473](https://issues.apache.org/jira/browse/AIRFLOW-6473)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/cli/commands/dag_command.py [hunk 1]
```diff
@@ -200,16 +200,22 @@ def dag_show(args):
...
+    >>> airflow dags state a_dag_with_conf_passed 2015-01-01T00:00:00.000000
+    failed, {"name": "bob", "age": "42"}
     """
     if args.subdir:
         dag = get_dag(args.subdir, args.dag_id)
     else:
         dag = get_dag_by_file_location(args.dag_id)
     dr = DagRun.find(dag.dag_id, execution_date=args.execution_date)
-    print(dr[0].state if len(dr) > 0 else None)  # pylint: disable=len-as-condition
+    out = dr[0].state if len(dr) > 0 else None  # pylint: disable=len-as-condition
```
Comment (feluelle) [inline airflow/cli/commands/dag_command.py]:
```suggestion
    out = dr[0].state if dr else None
```

Comment (tooptoop4) [reply airflow/cli/commands/dag_command.py]: fixed

**Comment by tooptoop4** [comment]
@feluelle pls merge

**Review by potiuk — APPROVED** [review]
Nice one! Thanks @tooptoop4 !

**Pull request merged**